recipes/tibi/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "tibi" %}
+{% set version = "1.1.0" %}
+{% set build = 0 %}
+{% set sha256 = "423b356a8ec0c356a9940c9d250686e82507f5ed5679f71d22f526ea0f968aeb" %}
+{% set python_min = "3.9" %}
```

Comment:
You only need to set this one if your min is higher than conda-forge's min.

```suggestion
```

Reply:
(addressed in cb9e70d4c478cb73eb577211bae33a587c63dc3d)